packages/jsActions/native-mobile-resources/src/platform/GetDeviceInfo.ts

```diff
@@ -17,8 +17,8 @@ export async function GetDeviceInfo(): Promise<mendix.lib.MxObject> {
         createMxObject("NativeMobileResources.DeviceInfo"),
         DeviceInfo.getBatteryLevel(),
         DeviceInfo.getFontScale(),
-        DeviceInfo.getFreeDiskStorage(),
-        DeviceInfo.getTotalDiskCapacity(),
+        DeviceInfo.getFreeDiskStorageOld(),
```

Comment:
Why are we using old ? In rnupgrades branch we are already using 6.0.0 and
![image](https://user-images.githubusercontent.com/7356299/98352769-bdc76080-201e-11eb-9e99-00b5766830d1.png)